### PR TITLE
fix: conda requirements

### PR DESCRIPTION
**Pull request by tadejsv**
This updates conda requirements, due to addition of `kubernetes` requirement

### CODE REVIEW

**Review by JoanFM — CHANGES_REQUESTED** [review]
Do we have docker images downloading the conda package for our executors? if not, I do not see the need for this. Kubernetes as default dependency should not be there for long

**Comment by JoanFM** [comment]
If there is no clear bug about this, I will close this PR. Kubernetes should soon disappear from our standard dependencies

**Comment by tadejsv** [comment]
@JoanFM yes there was a clear bug - conda and pip dependencies have to be in sync, otherwise conda packages do not build.

File: conda/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {% set name = "jina" %}
-{% set version = "2.0.23" %}
+{% set version = "2.1.5" %}
```
Comment (JoanFM) [inline conda/meta.yaml:2]: how is this adapted? conda versions will only exist for this hardcoded versions? we do not have any job to update this?

Comment (tadejsv) [reply conda/meta.yaml:2]: This is acutally auto-updated in the [feedstock repo](https://github.com/conda-forge/jina-feedstock) by conda-forge bot (see [this PR](https://github.com/conda-forge/jina-feedstock/pull/8) for example. The recipe here is not what is used by conda (you have to copy/paste it to feedstock repo first)

I could add some cd action here, but I feel it is not worth the effort - from 2.0.x till now I never had to make any adjustments, and I don't foresee this being a common thing. Right now I just need to make manual changes when there are changes in the requirements.
 
 What's really important is the script that generates the output.

**Review by JoanFM — APPROVED** [review]

**Pull request merged**